core/Tracker/Visitor.php

```diff
@@ -42,10 +42,10 @@ public function setVisitorColumn($column, $value)
         $this->visitProperties->setProperty($column, $value);
     }
 
-    public function getVisitorColumn($column)
+    public function getVisitorColumn($column, bool $original = false)
     {
-        if (array_key_exists($column, $this->visitProperties->getProperties())) {
-            return $this->visitProperties->getProperty($column);
+        if (array_key_exists($column, $this->visitProperties->getProperties($original))) {
+            return $this->visitProperties->getProperty($column, $original);
         }
```

Comment:
@bx80 could you please check this PR again. It still seems to contains some parts of the other PR that have been changed

Reply:
Must have missed that in the rebase, I've reverted it now. :+1: